modules/virt-cnv-ssp-alerts.adoc

```diff
@@ -0,0 +1,132 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-events.html/virt-virtualization-alerts.adoc
+
+[id="virt-cnv-ssp-alerts_{context}"]
+= SSP alerts
+
+SSP alerts provide information about problems for the CNV SSP Operator.
+
+//SSPFailingToReconcile Alert
+[id="SSPFailingToReconcile_{context}"]
+== SSPFailingToReconcile alert
+
+.Description
+
+The SSP Operator’s pod is up, but the pod's reconcile cycle consistently fails. This failure includes failure to update the resources it is responsible for, failure to deploy the template validator, or failure to deploy or update the common templates.
+
+.Reason
+
+If the SSP Operator fails to reconcile, then the deployment of dependent components fails, reconciliation of component changes fails, or both. Additionally, the updates to the common templates and template validator reset and fail.
+
+.Troubleshoot
+
+. Check the ssp-operator pod's logs for errors:
++
+[source,terminal]
+----
+export NAMESPACE="$(oc get deployment -A | grep ssp-operator | awk '{print $1}')"
+
+oc -n $NAMESPACE describe pods -l control-plane=ssp-operator
+
+oc -n $NAMESPACE logs --tail=-1 -l control-plane=ssp-operator
+----
+
+. Verify that the template validator is up. If the template validator is not up, then check the pod’s logs for errors.
++
+[source,terminal]
+----
+export NAMESPACE="$(oc get deployment -A | grep ssp-operator | awk '{print $1}')"
+
+oc -n $NAMESPACE get pods -l name=virt-template-validator
```

Comment:
oc -n $NAMESPACE get pods -l name=virt-template-validator --> this command might not result in anything. 

$ oc -n $NAMESPACE get pods -l name=virt-template-validator
No resources found in openshift-cnv namespace.


$ oc -n $NAMESPACE get  pods -l kubevirt.io=virt-template-validator
NAME                                       READY   STATUS    RESTARTS   AGE
virt-template-validator-6685b5f546-6d2jp   1/1     Running   0          37m
virt-template-validator-6685b5f546-br25p   1/1     Running   0          37m

Reply:
I think this works in lastest version the way you mentioned. I was checking in bit older version.
Please resolve this comment.

$ oc -n $NAMESPACE get pods -l name=virt-template-validator
NAME                                       READY   STATUS        RESTARTS   AGE
virt-template-validator-58645467cf-2lpz6   1/1     Terminating   0          20d
virt-template-validator-58645467cf-4jmdb   1/1     Running       0          12d
virt-template-validator-58645467cf-krjpl   1/1     Running       0          20d